### PR TITLE
Change openserach wording

### PR DESCRIPTION
**Pull request by hamza-m-masood**
## Description

Changed wording in `8.5` docs to include the opensearch guide on the install page.
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by christinaausley — DISMISSED** [review]

File: docs/self-managed/setup/install.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following charts will be installed as part of Camunda 8 Self-Managed:
   - _Note_: The chart is disabled by default and needs to be [enabled explicitly](#install-web-modeler) as Web Modeler is only available to enterprise customers.
 
 :::note Amazon OpenSearch Helm support
-The existing Helm charts use the Elasticsearch configurations by default and are not yet prepared with the OpenSearch configurations as templates/pre-filled. The Helm charts can still be used to install for OpenSearch, but some adjustments are needed beforehand.
+The existing Helm charts use the Elasticsearch configurations by default. The Helm charts can still be used to connect to AWS-managed OpenSearch. Refer to [using AWS-managed OpenSearch](/self-managed/setup/guides/using-existing-opensearch.md).
```
Comment (theburi) [inline docs/self-managed/setup/install.md:40]: One comment. The AWS product name is "Amazon OpenSearch Service". Could we update AWS-managed to Amazon OpenSearch Service so we are precise in the name?

Comment (christinaausley) [reply docs/self-managed/setup/install.md:40]: Great catch @theburi 👍

Comment (hamza-m-masood) [reply docs/self-managed/setup/install.md:40]: I would have to make this name change in the other docs as well. I can do that.

**Comment by hamza-m-masood** [comment]
@theburi @christinaausley please review again

**Review by theburi — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@theburi @hamza-m-masood Is this ready for merge?

**Comment by akeller** [comment]
Merging. Please open a new PR if you would like to make more changes.

**Pull request merged**